.goreleaser.yml

```diff
@@ -92,3 +92,5 @@ release:
   draft: true # we only publish after the Windows MSI gets uploaded
   prerelease: auto
   name_template: "MongoDB CLI {{.Version}}"
+  extra_files:
+    - glob: ./dist/"${MSI_FILE_NAME:?}".msi
```

Comment:
[q] what's the value of `msi_file_name`? also are you not able to use goreleaser template here? like using `{{.Version}}`? and let gorleaser handle the file name?

Reply:
The value of `msi_file_name` is `mongocli_windows_x86_64`

Reply:
why not use it here directly?
`./dist/mongocli_windows_x86_64.msi`

also I think it would be a step back to remove the version from the files we upload, it would be good to check if we can use go templates for this section

Reply:
and if you can't you can use globs so `./dist/*.msi` should work